xbmc/addons/PVRClient.cpp

```diff
@@ -375,35 +316,30 @@ bool CPVRClient::GetAddonProperties(void)
   CPVRTimerTypes timerTypes;
 
   /* get the capabilities */
-  PVR_ERROR retVal = m_struct.toAddon.GetAddonCapabilities(&addonCapabilities);
-  if (retVal != PVR_ERROR_NO_ERROR)
-  {
-    CLog::Log(LOGERROR, "PVR - couldn't get the capabilities for add-on '%s'. Please contact the developer of this add-on: %s", GetFriendlyName().c_str(), Author().c_str());
-    return false;
-  }
+  m_struct.toAddon.get_capabilities(&m_struct, &addonCapabilities);
```

Comment:
Sorry, if the API function can return error codes we should handle them here. The fact that the currently available addons never return an error is a coincedence and no guarantee that this will not change.